cmd/create/machinepool/machinepool.go

```diff
@@ -95,6 +95,7 @@ func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster
 	mpHelpers.HostedClusterOnlyFlag(r, cmd, "version")
 	mpHelpers.HostedClusterOnlyFlag(r, cmd, "autorepair")
 	mpHelpers.HostedClusterOnlyFlag(r, cmd, "tuning-configs")
+	mpHelpers.HostedClusterOnlyFlag(r, cmd, "kubelet-configs")
```

Comment:
Following the existing patterns as we are currently refactoring this in another effort and want to avoid a large refactor here.